graspy/inference/latent_distribution_test.py

```diff
@@ -366,18 +429,26 @@ def fit(self, A1, A2):
             X1_hat = check_array(A1)
             X2_hat = check_array(A2)
 
-        X1_hat, X2_hat = _median_sign_flips(X1_hat, X2_hat)
+        if self.align_type == "sign_flips":
+            aligner = SignFlips(**self.align_kws)
+            X1_hat = aligner.fit_transform(X1_hat, X2_hat)
+        if self.align_type == "seedless_procrustes":
+            aligner = SeedlessProcrustes(**self.align_kws)
+            X1_hat = aligner.fit_transform(X1_hat, X2_hat)
```

Comment:
1. should this be elif?
2. pull X1_hat out of the conditional?

Reply:
1. no practical difference, but i agree that elif looks better.
2. 👍

Reply:
cannot do 2: if you set to None - there is no aligner.

Reply:
fair